test/DynamoCoreTests/DynamoViewModelUnitTest.cs

```diff
@@ -18,6 +18,7 @@ namespace Dynamo.Tests
     public class DynamoViewModelUnitTest : UnitTestBase
     {
         protected DynamoViewModel ViewModel;
+        private DynamoShapeManager.Preloader preloader;
```

Comment:
Unit tests now rely on `DynamoShapeManager.Preloader` for its geometry preloading activities.